src/masternodes/rpc_proposals.cpp

```diff
@@ -739,6 +739,11 @@ UniValue votegovbatch(const JSONRPCRequest &request) {
         execTestTx(CTransaction(rawTx), targetHeight, optAuthTx);
 
         ret.push_back(signsend(rawTx, pwallet, optAuthTx)->GetHash().GetHex());
+        // Sleep the RPC worker thread a bit, so that the node can 
+        // relay the TXs as it works through. Otherwise, the main 
+        // chain can be locked for too long that prevent broadcasting of
+        // TXs
+        std::this_thread::sleep_for(std::chrono::seconds(1));
```

Comment:
1 second seems like a long time. Have we tested smaller time periods?

Reply:
- TL;DR: 20% overhead if your overall time is ~4h / (~<1h overhead, where count is still under a few hours) 
- But overhead becomes in-significant and exponentially lesser as the number or the time spent on signing grows. 
- Doing a corollary: if signing becomes much faster (with `-walletfastselect`, etc) `votegov` on a loop is still a way to get the job done faster. One or the other could cater to these.

Reply:
Switched to 0.5s in light of the perf. observations data with `-walletfastselect`